src/Okta.Sdk/Resource.cs

```diff
@@ -98,7 +97,7 @@ public T GetProperty<T>(string name)
                 return GetResourcePropertyInternal<T>(name);
             }
 
-            if (StringEnumTypeInfo.IsAssignableFrom(typeInfo))
+            if (typeof(T) == typeof(StringEnum))
```

Comment:
Why this change? (seems like it would exclude classes that inherit from `StringEnum`)